pkg/platform/runtime2/store/store.go

```diff
@@ -205,20 +209,25 @@ func (s *Store) Environ(inherit bool) (map[string]string, error) {
 	return envDef.GetEnv(inherit), nil
 }
 
-func (s *Store) UpdateEnviron() error {
+func (s *Store) UpdateEnviron(buildStatus *headchef_models.BuildStatusResponse) error {
```

Comment:
Nit: a little awkward that this is dealing with the full build response. Might be cleaner if we introduce a model function that just returns the data we want. Then it should also be much easier to unit test this.